### PR TITLE
Add project info for pypi

### PR DESCRIPTION
**Pull request by yngve-sk**

### CODE REVIEW

File: LICENSE [hunk 1]
```diff
@@ -0,0 +1,203 @@
...
+      License. However, in accepting such obligations, You may act only
+      on Your own behalf and on Your sole responsibility, not on behalf
+      of any other Contributor, and only if You agree to indemnify,
+      defend, and hold each Contributor harmless for any liability
+      incurred by, or claims asserted against, such Contributor by reason
+      of your accepting any such warranty or additional liability.
+
+   END OF TERMS AND CONDITIONS
+
+   APPENDIX: How to apply the Apache License to your work.
```
Comment (kwinkunks) [inline LICENSE]: I would not include the APPENDIX in the LICENSE file.

File: LICENSE [hunk 2]
```diff
@@ -0,0 +1,203 @@
...
+      To apply the Apache License to your work, attach the following
+      boilerplate notice, with the fields enclosed by brackets "[]"
+      replaced with your own identifying information. (Don't include
+      the brackets!)  The text should be enclosed in the appropriate
+      comment syntax for the file format. We also recommend that a
+      file or class name and description of purpose be included on the
+      same "printed page" as the copyright notice for easier
+      identification within third-party archives.
+
+   Copyright 2013 National Renewable Energy Laboratory (NREL)
```
Comment (kwinkunks) [inline LICENSE]: I would remove the Copyright notice too and instead put it in the source files, OR (as a quick shortcut) in 1 conspicuous place in `src`

File: docs/Apache2.0License.txt
Comment (kwinkunks) [inline docs/Apache2.0License.txt:1]: Good idea to remove this.

**Comment by kwinkunks** [comment]
Did you also update `src/Apache2.0License.txt`?

I would also remove the copyright notice from `setup.py`

I don't see Equinor/Statoil in _any_ of the source files... do we know exactly what we have updated? 

NB There should strictly be a `NOTICE` file mentioning the changes we made, according to the licence.

**Review by kwinkunks — COMMENTED** [review]
I think you will want someone more closely related to the project to _approve_ this, but from the point of view of clarity around licensing I think you've done a great job of improving things here 👍

**Comment by oysteoh** [comment]
@sondreso is this PR fine from your point of view or anything more that should be done? I think it is ready  for merging....

**Review by sondreso — APPROVED** [review]
I think this looks very reasonable!

**Pull request merged**